cloudferrylib/os/actions/transport_storage_resources.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2016 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import log
+from cloudferrylib.utils import utils as utl
+
+
+LOG = log.getLogger(__name__)
+
+
+class TransportStorageResources(action.Action):
+    """This action is for transporting cinders'
+    resources(still quotas) from src on dst.
+
+    Process:
+     1. Read info about resources
+     2. Deploy resources on dst.
+
+     No specific config parameters.
+     No dependence on other actions.
```

Comment:
I'd say `migrate_volumes` action actually depends on quotas, so I'd recommend adding it here